crates/sui-graphql-rpc/src/types/stake.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::*;
+
+pub(crate) struct Stake;
+pub(crate) struct StakeConnection;
+
+#[allow(unreachable_code)]
+#[allow(unused_variables)]
+#[Object]
+impl Stake {
+    async fn id(&self) -> ID {
+        unimplemented!()
+    }
+}
+
+#[allow(unreachable_code)]
+#[allow(unused_variables)]
+#[Object]
+impl StakeConnection {
+    async fn id(&self) -> ID {
```

Comment:
Do the connections themselves need `id`?

Reply:
indeed this is not needed. But the graphql lib will complain if a type is defined without any fields/methods. I initially put these as a placeholder but should remove them to avoid confusion.